game/scripts/npc/items/custom/item_martyrs_mail_4.txt

```diff
@@ -27,7 +27,7 @@
     {
       "01"                                                "item_spiked_mail_4;item_upgrade_core_4"
       "02"                                                "item_shivas_guard_4;item_upgrade_core_4"
-      "03"                                                "item_stoneskin;item_upgrade_core_4"
+      //"03"                                                "item_stoneskin;item_upgrade_core_4"
       "04"                                                "item_assault_4;item_upgrade_core_4"
```

Comment:
gotta change this to 03 so that it has 01 02 03 instead of 01 02 04. causes weird subtle bugs in the shop menu otherwise

Reply:
i have done